src/components/SeoForm/SeoForm.tsx

```diff
@@ -168,11 +261,8 @@ const SeoForm: React.FC<SeoFormProps> = props => {
                   )}
                 </div>
               }
-              helperText={intl.formatMessage({
-                defaultMessage:
-                  "If empty, the preview shows what will be autogenerated."
-              })}
-              value={description ? description.slice(0, 299) : undefined}
+              helperText={intl.formatMessage(seoFieldMessage)}
```

Comment:
If `getError(SeoField.description) === true`, this input will be red but won't display any message

Reply:
Ok, removing until we get errors for those fields from backend